src/Avalonia.Base/Media/Fonts/FontCollectionBase.cs

```diff
@@ -80,6 +81,47 @@ IEnumerator IEnumerable.GetEnumerator()
             return GetEnumerator();
         }
 
+        protected bool TryCreateSyntheticGlyphTypeface(IGlyphTypeface glyphTypeface, FontStyle style, FontWeight weight,
```

Comment:
Should probably be `private protected` so it isn't part of the public API.

Reply:
I have moved that method to FontManger and made it internal